testing/toxcore/APKBUILD

```diff
@@ -0,0 +1,48 @@
+# Maintainer: Jonathan Sieber <mail@strfry.org>
+
+pkgname=toxcore
+pkgver=0.2.2
+pkgrel=0
+pkgdesc="Tox communication project - Core Library"
+url="https://tox.chat/"
+arch="all"
+license="GPL"
+makedepends="linux-headers cmake libsodium-dev opus-dev libvpx-dev"
+subpackages="$pkgname-dev $pkgname-doc"
+source="$pkgname-$pkgver.tar.gz::https://github.com/TokTok/c-toxcore/archive/v$pkgver.tar.gz"
+
+# Toxcore's tests do require networking and are not working properly in Travis-CI
+options="!check"
+
+builddir="$srcdir/c-$pkgname-$pkgver"
+
+prepare() {
+	cd "$builddir"
+	cmake . \
```

Comment:
I think @clandmeter was trying to explain that `cmake` should be called inside the `build()` function, once you've moved the `cmake` command I don't think there is any need for a `prepare()` function at all you can probably remove the whole function definition and rely on abuild running the default automatically